src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,80 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarsService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Manufacturer audiManufacturer = new Manufacturer("Audi", "Germany");
+        Manufacturer fiatManufacturer = new Manufacturer("Fiat", "Italy");
+        Manufacturer renaultManufacturer = new Manufacturer("Renault", "France");
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        manufacturerService.create(audiManufacturer);
+        manufacturerService.create(fiatManufacturer);
+        manufacturerService.create(renaultManufacturer);
+        System.out.println("get manufacturer with id = " + audiManufacturer.getId() + " --- "
+                + manufacturerService.get(audiManufacturer.getId()));
+        System.out.println("get all manufacturers --- " + manufacturerService.getAll());
+        Manufacturer toyotaManufacturer = new Manufacturer(2L, "Toyota", "Japan");
```

Comment:
why are you explicitly setting an id here? Also let's remove all hardcoded ids from your code and get it using getId() call of the corresponding model